src/cashier/withdraw.es6

```diff
@@ -7,12 +7,11 @@ import liveapi from 'websockets/binary_websockets';
 import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import currencyDialog from 'cashier/currency';
-import _ from 'lodash';
+import {debounce} from 'lodash'
```

Comment:
@4p00rv I figured out why it is sending separate requests for imported module. @aminroosta  is using `transform-es2015-modules-amd`. I am not sure if I would like to continue using this, but changing this would require some effort. For now I will let it go and when we move completely to webpack, we can get rid of this AMD based babel plugin. webpack supports dividing code into `chunks`.